components/stash/stash.html

```diff
@@ -10,8 +10,8 @@ <h4 class="stash-toggle-text" data-bind="click: toggleShowStash">
     </h4>
     <div class="list-group" data-bind="foreach: stashedChanges">
       <div class="list-group-item" data-ta-container="stash-stash">
-        <a href="#" data-bind="click: pop" class="glyphicon glyphicon-pencil glyphicon-circled pull-left" data-ta-clickable="stash-pop"></a>
-        <a href="#" data-bind="click: toggleShowCommitDiffs" data-ta-clickable="stash-diff">
+        <a href="#" data-bind="click: pop" class="glyphicon glyphicon-pencil glyphicon-circled pull-left" data-ta-clickable="stash-pop" data-toggle="tooltip" title="Apply this stash"></a>
```

Comment:
Good point. It is poping the stash.
I thought it was the same by looking at the documentation.
`git stash ( pop | apply ) [--index] [-q|--quiet] [<stash>]`

But it isn't http://stackoverflow.com/a/15286090